src/components/Thumbnail/Description.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { FC } from "react";
+import classnames from "classnames";
+
+export interface IProps {
+  classname?: string;
+}
+
+export const Description: FC<IProps> = ({ children, classname = "" }) => (
+  <p className={classnames("text-sm font-lato", classname)}>{children}</p>
```

Comment:
we need handle long texts, use text-overflow: ellipsis;
https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

Reply:
ellipsis is not supported on tailwind, had to use "whitespace-normal overflow-hidden"

Reply:
tailwind has a special `truncate` class for this, find it here: https://tailwindcss.com/docs/word-break/#app